docker-compose.metrics.yml

```diff
@@ -0,0 +1,26 @@
+version: '2.1'
+services:
+  graphite:
+    image: sitespeedio/graphite
+    ports:
+        - "8181:80"
+
+  statsite:
+    image: quay.io/azavea/statsite:0.7.1
```

Comment:
Yeah definitely go for it. I figured there'd be a bunch of edits to this PR - wanted to get things started though.